test/test_site.rb

```diff
@@ -192,12 +192,14 @@ def generate(site)
         index.html
         index.html
         main.scss
-        main.scss
         properties.html
         sitemap.xml
         static_files.html
-        symlinked-file
       )
+      unless Jekyll::Utils::Platforms.windows?
```

Comment:
You can choose whether to `skip_when_not_windows`, dealers choice on that, considering the test will pass on Linux as well, so... yeah.